sdk/identity/azure_identity/README.md

```diff
@@ -1,77 +1,425 @@
+
+### Provide feedback
+
+If you encounter bugs or have suggestions, [open an issue](https://github.com/Azure/azure-sdk-for-rust/issues).
+
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit [https://cla.microsoft.com](https://cla.microsoft.com).
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You'll only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information, see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+<!-- LINKS -->
+[azure_cli]: https://learn.microsoft.com/cli/azure
+[azure_data_cosmos]: https://github.com/Azure/azure-sdk-for-rust/tree/main/sdk/cosmos/azure_data_cosmos
+[azure_developer_cli]: https://learn.microsoft.com/azure/developer/azure-developer-cli/
+[azure_sub]: https://azure.microsoft.com/free/
+[cert_cred_ref]: <!-- TODO: When Docs.rs page for ClientCertificateCredential ref docs are available -->
+[cli_cred_ref]: <!-- TODO: When Docs.rs page for AzureCliCredential ref docs are available>
+[dac_overview]: <!-- TODO: When we have a conceptual doc on Credential chains with a section on DefaultAzureCredential overview. Python example: https://learn.microsoft.com/azure/developer/python/sdk/authentication/credential-chains?tabs=dac#defaultazurecredential-overview -->
+[default_cred_ref]: <!-- TODO: When Docs.rs page for DefautlAzureCredential ref docs are available -->
+[entraid_doc]: https://learn.microsoft.com/entra/identity/
+[identity_api_docs]: https://docs.rs/azure_identity/latest/azure_identity/
+[managed_id_cred_ref]: <!-- TODO: When Docs.rs page for ImdsManagedIdentityCredential ref docs are available -->
+[package]: https://crates.io/crates/azure_identity
+[source]: https://github.com/Azure/azure-sdk-for-rust/tree/main/sdk/identity/azure_identity
+[token_cred_ref]: <!-- TODO: When Docs.rs page for TokenCredential trait ref docs are available -->
+[workload_id_cred_ref]: <!-- TODO: When Docs.rs page for WorkloadIdentityCredential ref docs are available -->
+
+<!-- LINKS from Python Example for reference>
```

Comment:
We are all these here? We can always copy what we need later and update the URLs. That's the great thing about source control: we never really lose anything anyway.

Reply:
Removed